ci-operator/config/openshift/knative-eventing/openshift-knative-eventing-release-v0.14.2.yaml

```diff
@@ -9,6 +9,9 @@ build_root:
     dockerfile_path: openshift/ci-operator/build-image/Dockerfile
 canonical_go_repository: knative.dev/eventing
 images:
+- dockerfile_path: openshift/ci-operator/source-image/Dockerfile
+  from: src
+  to: knative-eventing-src
```

Comment:
I think there's now another way to specify the image - easier:
```
promotion:
  additional_images:
    knative-eventing-src: "src" 
```
Coud you try it in this PR please?

Reply:
Found out just today.